### PR TITLE
Oui non

### PR DESCRIPTION
**Pull request by Iluvrog**
Add a yes/no game in the opinions game option

### CODE REVIEW

**Review by PillouMatou — CHANGES_REQUESTED** [review]
Translate your buttons
image
- maybe remove the bar in the middle of the screen

**Comment by Iluvrog** [comment]
> Translate your buttons
> image
> 
> * maybe remove the bar in the middle of the screen

translation done and bar remove

**Review by PillouMatou — APPROVED** [review]
it's fine for me @kuhlkrein @schwabdidier

File: gazeplay-games/src/main/java/net/gazeplay/games/opinions/OpinionsGame.java [hunk 1]
```diff
@@ -92,63 +101,113 @@ public void launch() {
...
+
+            thumbUp.assignIndicator(event -> {
+                background.setFill(new ImagePattern(backgroundImage.pickRandomImage()));
+                stats.incrementNumberOfGoalsReached();
+                updateScore();
+            }, configuration.getFixationLength());
+            gameContext.getGazeDeviceManager().addEventFilter(thumbUp);
+            thumbUp.active();
+
+            /*List<Image> Picture = thumbImage.pickAllImages();
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/opinions/OpinionsGame.java:139]: please remove the useless comments here

File: gazeplay-games/src/main/java/net/gazeplay/games/opinions/OpinionsGame.java [hunk 2]
```diff
@@ -92,63 +101,113 @@ public void launch() {
...
+                createAddButtonOpinions(Oui, "data/opinions/thumbs/correct2.png", dimension2D.getWidth() / 2 - dimension2D.getWidth() / 20, 0);
+                createAddButtonOpinions(Non, "data/opinions/thumbs/error.png", dimension2D.getWidth() / 2 - dimension2D.getWidth() / 20, dimension2D.getHeight() * 16 / 20);
             }
-            if (I.getUrl().equals("file:/C:/Users/MATOU/GazePlay/files/images/opinions/thumbs/thumbup.png")) {
-                ImageView thumbU = new ImageView(new ImagePattern(new Image("file:/C:/Users/MATOU/GazePlay/files/images/opinions/thumbs/thumbup.png")).getImage());
-                thumbUp.setImage(thumbU);
-                thumbU.setFitWidth(dimension2D.getWidth() / 10);
-                thumbU.setFitHeight(dimension2D.getHeight() / 5);
+
+            if (type.equals(OpinionsGameVariant.ONBH)) {
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/opinions/OpinionsGame.java:177]: Please use 'else if' instead of only if to avoid multiple tests

**Review by kuhlkrein — CHANGES_REQUESTED** [review]

**Comment by kuhlkrein** [comment]
Somme changes are needed and there are some conflicts in files too :)

**Comment by Iluvrog** [comment]
> Some changes are needed and there are some conflicts in files too :)

It's done

**Review by kuhlkrein — APPROVED** [review]
Looks good now

**Pull request merged**